From 95821fc46a7084c14ece6397d58efb486d1264fc Mon Sep 17 00:00:00 2001
From: Pankaj Dev <pankaj.dev@st.com>
Date: Fri, 26 Jan 2024 12:19:14 +0530
Subject: [PATCH 0923/1141] arm64: defconfig: Enable ONBOARD USB Hub driver

USB Hub mounted on multiple stm32mp25 boards like eval, dk
Enabling driver for onboard-usb-hub

Change-Id: I79f50be7d233ce8d19cf81d8478cde13f4ca0000
Signed-off-by: Pankaj Dev <pankaj.dev@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/354577
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -109,6 +109,7 @@ CONFIG_UBIFS_FS=y
 CONFIG_UCSI_STM32G0=m
 CONFIG_USB_DWC3_STM32=y
 CONFIG_USB_HIDDEV=y
+CONFIG_USB_ONBOARD_HUB=y
 CONFIG_USB_STM32_USBH=y
 CONFIG_V4L_TEST_DRIVERS=y
 CONFIG_VIDEOMODE_HELPERS=y
